src/core/struct_tree.js

```diff
@@ -281,6 +282,12 @@ class StructTreeRoot {
     }
   }
 
+  static convertString(str) {
+    return isAscii(str)
+      ? str
+      : stringToUTF16String(str, /* bigEndian = */ true);
+  }
```

Comment:
The same code seems to be duplicated *multiple* times already in https://github.com/mozilla/pdf.js/blob/master/src/core/annotation.js, so can we please add a new helper function (placed in `/src/core/core_utils.js`) and use that everywhere instead?

Reply:
Definitely, you've a way better memory than me... I'm a bit jealous :).